### ISSUE TITLE
Where-Used feature to determine where global widgets and templates are being used

### ISSUE THREAD

**Issue by ShadowXVII** [elementor/elementor#12824]
## Prerequisites

- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
I've built up a library of sections and global widgets that are used throughout designs
However, this list becomes littered with sections that might not be used anymore, i.e. shortcodes embedded in places that no longer need them.

**Describe the solution you'd like**
A way to identify how many pages/where a particular Template or shortcode is being used. This doesn't have to be LIVE, i.e. a button to "calculate" usage would be OK.

**Describe alternatives you've considered**
Letting the Template library fill with un-used templates.

**Comment by black-eye** [comment]
+1

**Comment by ErwinAI** [comment]
I'd like this as well.

**Comment by Mij-Strebor** [comment]
Indeed, some sort of cross-reference listing of components is necessary for security and load time.

**Comment by zft9xgy** [comment]
+1, any news in this feat?

**Comment by clmnsm** [comment]
+1
